Src/PerformanceCollector/Unit.Tests/QuickPulse/QuickPulseServiceClientTests.cs

```diff
@@ -647,6 +680,59 @@ public void QuickPulseServiceClientSubmitsIsWebAppToService()
             Assert.IsTrue(this.samples[0].Item2.IsWebApp);
         }
 
+        [TestMethod]
+        public void QuickPulseServiceClientSubmitsTopCpuProcessesToService()
+        {
+            // ARRANGE
+            var now = DateTimeOffset.UtcNow;
+            var ikey = "some ikey";
+            var serviceClient = new QuickPulseServiceClient(this.serviceEndpoint, string.Empty, string.Empty, string.Empty, string.Empty, new Clock(), true);
+            var cpuData = new[] { Tuple.Create("Process1", 1), Tuple.Create("Process2", 2) };
+
+            var sample = new QuickPulseDataSample(
+                new QuickPulseDataAccumulator { StartTimestamp = now, EndTimestamp = now.AddSeconds(1) },
+                new Dictionary<string, Tuple<PerformanceCounterData, double>>(),
+                cpuData,
+                false);
+
+            // ACT
+            serviceClient.SubmitSamples(new[] { sample }, ikey);
```

Comment:
So is this really an integration test that makes real calls rather than a unit test or is the service client mocked in some way?